MapEditor.py

```diff
@@ -188,6 +194,23 @@ def _get_paths(self):
             title="Select a directory to place the patch.")
         self.settings.write(self.paths)
 
+    # region public functions
+
+    def return_edit(self, event):
```

Comment:
method name doesn't really make sense... same with `return_decompile`

Reply:
I can update it if you want. I was going off of the fact that it is triggered via the `<Return>` keybind. Any suggestions?

Reply:
either something like `call_decompile`, or just have it a public `decompile` function which then calls `_decompile`. That way if you want to tack extra things on you can add them to the `decompile` function

Reply:
Alright, I've updated their names to `edit` and `decompile`.